content_scripts/link_hints.coffee

```diff
@@ -384,7 +384,17 @@ class LinkHintsMode
             window.focus()
             DomUtils.simulateSelect clickEl
           else
-            clickActivator = (modifiers) -> (link) -> DomUtils.simulateClick link, modifiers
+            clickActivator = (modifiers) -> (link) ->
+              defaultActionsTriggered = DomUtils.simulateClick link, modifiers
+              simulateClickDefaultAction = Utils.isFirefox()
+              if simulateClickDefaultAction and
```

Comment:
@mrmr1993.  We can merge this.  But I'd really appreciate it if you could find a way of avoiding repeating the modifier/new-tab logic here.  That logic rightly belongs -- in my opinion -- with the various modes at the top of this file.

Reply:
What do you think about moving it into `DomUtils.simulateClick`? Since it's a workaround for that rather than specifically for link hints, there makes more sense to me.

I'll rebase and make the change -- whichever you choose -- in a little bit.